fastlane/Fastfile

```diff
@@ -13,7 +13,8 @@ files_to_update_phc_version = {
     'react-native-purchases-ui/RNPaywalls.podspec' => ['"PurchasesHybridCommon", \'{x}\''],
     'android/build.gradle' => ['com.revenuecat.purchases:purchases-hybrid-common:{x}'],
     'react-native-purchases-ui/android/build.gradle' => ['com.revenuecat.purchases:purchases-hybrid-common-ui:{x}'],
-    'package.json' => ['"@revenuecat/purchases-typescript-internal": "{x}"']
+    'package.json' => ['"@revenuecat/purchases-typescript-internal": "{x}"'],
+    'yarn.lock' => ['"@revenuecat/purchases-typescript-internal": {x}']
```

Comment:
Als, I wonder if we should be committing the yarn.lock file... seems that it should be enough with the package.json?

Reply:
I think we should, that way the `yarn` command is deterministic https://yarnpkg.com/getting-started/qa#should-lockfiles-be-committed-to-the-repository

Reply:
Regarding this change... I don't think we should modify the `yarn.lock` file like this and we should run `yarn install` to modify the yarn.lock accordingly